dev/tests/test_update_ml_package_versions.py

```diff
@@ -22,17 +23,23 @@ def __exit__(self, exc_type, exc_value, traceback):
         pass
 
     @classmethod
-    def from_versions(cls, versions):
+    def from_versions(cls, versions, uploaded_year_diff_list=None):
```

Comment:
why do we need `uploaded_year_diff_list`? Can pass both version and date to `versions`?